models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -158,22 +154,22 @@ LEFT JOIN {{ ref('tokens_erc20') }} erc20b
     AND erc20b.blockchain = 'avalanche_c'
 LEFT JOIN {{ source('prices', 'usd') }} p_bought
     ON p_bought.minute = date_trunc('minute', kyberswap_dex.block_time)
-    AND p_bought.contract_address = kyberswap_dex.token_bought_address
+    AND cast(p_bought.contract_address as varbinary) = kyberswap_dex.token_bought_address
     AND p_bought.blockchain = 'avalanche_c'
     {% if is_incremental() %}
-    AND p_bought.minute >= date_trunc("day", now() - interval '1 week')
+    AND p_bought.minute >= date_trunc('day', now() - interval '7' day)
     {% else %}
-    AND p_bought.minute >= '{{project_start_date}}'
+    AND p_bought.minute >= TIMESTAMP '{{project_start_date}}'
     {% endif %}
 LEFT JOIN {{ source('prices', 'usd') }} p_sold
     ON p_sold.minute = date_trunc('minute', kyberswap_dex.block_time)
-    AND p_sold.contract_address = kyberswap_dex.token_sold_address
+    AND cast(p_sold.contract_address as varbinary) = kyberswap_dex.token_sold_address
```

Comment:
is this cast necessary? the field should be varbinary already, if not, that is an issue

Reply:
Yeah actually I was surprised too but was getting an error about the `contract_address` column on prices_usd being varchar. Could this be something that is happening on Dev only, or a problem with my setup?

Reply:
I will remove the cast as it sounds like something that should be working properly and not something I try to cast as a way to pass tests.